tutorials/plot_measures.py

```diff
@@ -11,40 +11,124 @@
 # Applying measures & conversions to spiking data
 # -----------------------------------------------
 #
-# Words, words, words.
+# Sections
+# ~~~~~~~~
+#
+# This tutorial contains the following sections:
+#
+# 1. Compute measures of spiking activity
+# 2. Convert spiking data
 #
 
 ###################################################################################################
 
 # sphinx_gallery_thumbnail_number = 1
 
-import spiketools
+# import auxiliary libraries
+import numpy as np
+import matplotlib.pyplot as plt
+
+# import all functions from spiketools.measures
+from spiketools.measures.measures import *
+from spiketools.measures.conversions import *
 
 ###################################################################################################
 
-# Settings
-...
+# Compute measures of spiking sctivity
+# ~~~~~~~~
```

Comment:
For title headings (things like the `~` here), the underlining should go as long as the title, so something like:

```
# Compute measures of spiking sctivity
# ~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Also, quick sidenote: type on `sctivity`